src/RAM/Science/Science.tsx

```diff
@@ -76,6 +78,10 @@ function science(): void {
       rovecomm.sendCommand('MicroscopeFocus', -microscopeMult);
     }
   }
+
+  if ('DropSample' in controllerInputs) {
+    rovecomm.sendCommand('ScoopGrabber', 2);
+  }
```

Comment:
This never checks for if the button is pressed, only if the config for it exists. You should have a nested if statement checking for `controllerInputs.DropSample === 1`